src/main/java/com/everest/employeeportal/controllers/EmployeeController.java

```diff
@@ -2,28 +2,28 @@
 
 
 import com.everest.employeeportal.entities.Employee;
-import com.everest.employeeportal.repositories.EmployeeRepositoryInMemory;
+import com.everest.employeeportal.services.EmployeeService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.List;
+import java.util.Collection;
 
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("v1/api/employees")
 public class EmployeeController {
-    private final EmployeeRepositoryInMemory employeeRepository;
+    private final EmployeeService employeeService;
 
     @GetMapping(value = "")
-    public List<Employee> getEmployees(){
-        return employeeRepository.getEmployees();
+    public Collection<Employee> getEmployees(){
```

Comment:
This is not the format we are expecting. Please check the documentation.

Reply:
I've changed this format in the other branch i.e _feat-add-pagination_ .